src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.Windows.cs

```diff
@@ -26,6 +26,19 @@ internal static CultureInfo GetUserDefaultCulture()
             return GetCultureByName(strDefault);
         }
 
+        private static CultureInfo GetPredefinedCultureInfo(string name)
+        {
+            CultureInfo culture = GetCultureInfo(name);
+            string englishName = culture.EnglishName;
+
+            if (englishName.StartsWith("Unknown Locale", StringComparison.Ordinal) || englishName.StartsWith("Unknown Language", StringComparison.Ordinal))
```

Comment:
Nit: you might consider minimizing the need to do two StartsWith in the common case by first doing a StartsWith just for "Unknown "